parser.py

```diff
@@ -119,23 +145,23 @@ def parse_round(bsoup, sql, rnd, gid, airdate):
 
 def insert(sql, clue):
     """Inserts the given clue into the database."""
-    # Clue is [game, airdate, round, category, value, clue, answer]
+    # Clue is [game, airdate, game_comments, game_type, round, category, value, clue, answer]
     # Note that at this point, clue[4] is False if round is 3
```

Comment:
This comment should read: `# Note that at this point, clue[6] is False if round is 3`